routerchain/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "routerchain",
+    "assets": [
+      {
+        "description": "The native token of Router Chain",
+        "extended_description": "Router Chain is a PoS L1 that allows stateful bridging–enabling truly omnichain dapps that can abstract chains from the users and aggregate liquidity across multiple chains.",
+        "denom_units": [
+          {
+            "denom": "route",
+            "exponent": 18
+          }
+        ],
+        "base": "route",
+        "name": "Router Chain",
+        "display": "route",
+        "symbol": "ROUTE",
+        "coingecko_id": "router-protocol",
```

Comment:
As Per https://api.coingecko.com/api/v3/coins/list, the coingecko-id is just "route"